CHANGELOG.md

```diff
@@ -8,7 +8,11 @@
 
 ### Added
 
-* Support for `DATABASE_URL` values that use the `mariadb` scheme. ([#341](https://github.com/heroku/heroku-buildpack-jvm-common/pull/341)
+* Support for `DATABASE_URL` values that use the `mariadb` scheme. ([#341](https://github.com/heroku/heroku-buildpack-jvm-common/pull/341))
+
+### Removed
+
+* `export_env_dir`, `copy_directories`, `curl_with_defaults` and `nowms` functions from `bin/util`. ([#344](https://github.com/heroku/heroku-buildpack-jvm-common/pull/344))
```

Comment:
This presumably will break some third-party consumers of jvm-common as a library?

One option might be to leave shim functions in `bin/util` that emit deprecation warnings for a bit.

Though I'm also fine with making this change and handling any third-party updates as they come up.

Reply:
Or even perhaps a `bin/util` that emits an error message and then `exit 1`s, so if anyone is still sourcing it they get a clear error rather than "no such file" etc.

Reply:
I will address this in a follow-up PR. :)